addons/mail/static/src/core/web/activity.js

```diff
@@ -42,6 +42,13 @@ export class Activity extends Component {
         return this.props.activity.display_name;
     }
 
+    get note() {
+        return (
+            this.props.messageSearch?.highlight(this.props.activity.note) ??
+            this.props.activity.note
```

Comment:
highlighting is only done for note but not for other searchable fields?

Reply:
The only other field would be the `displayName`, which depends on whether the activity contains a summary or not.
I tried implementing the highlighting on that field but the `displayName` string gets displayed with the `t-esc` directive and since the highlighting adds a `span` tag it would not work.
For some reason the `note` field gets escaped before and shown with `t-out` but the `summary` field does not get escaped before.